mp/src/game/client/momentum/ui/HUD/hud_concgrenade.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <vgui_controls/EditablePanel.h>
+#include <vgui_controls/Label.h>
+#include <vgui_controls/ProgressBar.h>
+
+#include "hudelement.h"
+
+#include "mom_concgrenade.h"
+#include "weapon/weapon_mom_concgrenade.h"
+
+class CHudConcTimer : public CHudElement, public vgui::EditablePanel
+{
+    DECLARE_CLASS_SIMPLE(CHudConcTimer, EditablePanel);
+
+    CHudConcTimer(const char *pElementName);
+
+    bool ShouldDraw() OVERRIDE;
+    void OnThink() OVERRIDE;
+    void Reset() OVERRIDE;
+
+  private:
+    CMomentumConcGrenade *m_pGrenade;
+    vgui::ContinuousProgressBar *m_pTimer;
+    vgui::Label *m_pTimerLabel;
+};
+
+DECLARE_HUDELEMENT(CHudConcTimer);
+
+class CHudConcEntPanel : public vgui::Panel
+{
+    DECLARE_CLASS_SIMPLE(CHudConcEntPanel, vgui::Panel);
+
+    CHudConcEntPanel();
+    ~CHudConcEntPanel();
+
+    void Init(CMomConcProjectile *pEntity);
+    void OnThink() OVERRIDE;
```

Comment:
Not sure if this really matters but i see the old override macro used throughout this and the other conc code.